### PR TITLE
Dockerfile  Added and Yarn audit result will be updated in issue #29 

### LINKED ISSUE #29: Add Dockerfile

### PR DESCRIPTION
**Pull request by evalsocket**
Resolves #29

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# Stage 1: Build container
+FROM node:lts-alpine as builder
+
+# Install dependencies
+# Git and python are currently necessary to install some packages
+RUN apk update
+RUN apk upgrade
+COPY package*.json ./
+RUN apk add --no-cache --virtual .gyp git python make g++ bash
+RUN npm install
```
Comment (aphelionz) [inline Dockerfile]: Note the critical error. Let's definitely add `RUN npm audit fix` here. 
screenshot_20190107_164507

Comment (aphelionz) [reply Dockerfile]: @evalsocket I approved this by accident so please fix this before we merge

Comment (aphelionz) [reply Dockerfile]: Actually I just realized that there is no package-lock.json in the repo. Forget about adding `npm audit fix,` just update package.json and commit package-lock.json.

Comment (evalsocket) [reply Dockerfile]: Can you review it again i just change pkg manager from npm to yarn 
Please check [logs](https://hackmd.io/s/rJf4cU-G4)

Comment (aphelionz) [reply Dockerfile]: What advantages does changing from `npm` to `yarn` give us? I think we just need an updated package.json that doesn't have vulnerabilities while running `docker build` and we'll be good to go. 

We can also do that in another PR if it suits you.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Use latest stable (LTS) alpine releases
+
+# Stage 1: Build container
+FROM node:lts-alpine as builder
+
+# Install dependencies
+# Git and python are currently necessary to install some packages
+RUN apk update
```
Comment (vvp) [inline Dockerfile]: Can you also inline these apk-commands into oneliner and clean the caches after? Having smaller images and less layers to cache is friendlier to the users :slightly_smiling_face:

Comment (evalsocket) [reply Dockerfile]: ok

File: Dockerfile (hunk 1 above)
Comment (evalsocket) [reply Dockerfile]: npm 5 is also faster so there is no point of discussion i revert the changes and in package-lock.json is added in repo.
**Npm audit result** -  There is a critical vulnerabilities command line injection in grunt-open@0.2.3 and no one is maintaining it. Some forks replace grunt-open by open. I think we can go with it for docker and create a new issue for removing warnings and vulnerabilities from package.json

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# Use latest stable (LTS) alpine releases
+
+# Stage 1: Build container
+FROM node:lts-alpine as builder
+
+# Install dependencies
+# Git and python are currently necessary to install some packages
+RUN apk update && apk upgrade  && apk add --no-cache --virtual .gyp git python make g++ bash
+COPY package*.json ./
+RUN npm
```
Comment (aphelionz) [inline Dockerfile]: Oops, should be `RUN npm install`

**Comment by aphelionz** [comment]
@evalsocket LGTM! :+1:

**Pull request merged**